### ISSUE TITLE
Cloud Agent to publish PrismDID on behalf of external wallet into VDR

### ISSUE THREAD

**Issue by robertocarvajal** [hyperledger/identus#33]
**Is your feature request related to a problem? Please describe.**
Currently, Cloud Agent can only publish PrismDIDs that are created by the same Cloud Agent into the VDR. Most Edge Clients will never be connected to a full node in order to publish their PrismDIDs to the VDR and it could be useful for certain use cases.

**Describe the solution you'd like**
It would be great to ask the Cloud Agent to publish a PrismDID created from my Edge Client Wallet through the REST API.

**Describe alternatives you've considered**
Allowing the Edge SDK to connect directly to icarus (like the cloud agent does) and prepare the right payload which I would need to reverse engineer anyways from the cloud agent code :)

**Comment by FabioPinheiro** [comment]
#86 is also related with this 'need'.
But the goal in there is to bypass the Cloud Agent + Prism Node to publish the `did:prism`

**Comment by robertocarvajal** [comment]
yeah, I feel we can close this issue and keep the discussion over the other one because it has a broader scope and was discussed recently over the weekly meetings.

The end goal is the same, put a DID on chain that was created by the edge SDK.
